chain/store/splitstore/snoop.go

```diff
@@ -0,0 +1,161 @@
+package splitstore
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/bmatsuo/lmdb-go/lmdb"
+
+	cid "github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/go-state-types/abi"
+)
+
+var TrackingStoreMapSize int64 = 1 << 34 // 16G
```

Comment:
mmapping 16GiB might be quite a bit for this store. We should really implement step-wise incremental mmapping like bbolt. https://github.com/etcd-io/bbolt/blob/f6be82302843a215152f5a1daf652c1ee5503f85/db.go#L16-L17

Reply:
dunno, I just set it to 16G to be safe, it shouldn't really matter.

We can tackle incremental mapping on a later pr.

Reply:
Please add it to the TODO then

Reply:
ok. It needs to happen in the blockstore implementation as well, where we map a gigantic 256G file.

Reply:
I added a TODO note.